### ISSUE TITLE
If there is a single SLC left in a mini-stack Fringe crashes

### ISSUE THREAD

**Issue by jlmaurer** [isce-framework/fringe#19]
Specific Example: 
- 121 total SLCs 
- mini-stack of size 20
- leaves a single SLC
Result: 
- no EVD folder created for the single SLC
- integratePS.py crashes because it can't find the EVD folder for the last SLC 
Possible work-arounds: 
- just drop the extra one
- take one from the previous mini-stack to make a mini-stack of 2?

**Comment by piyushrpt** [comment]
Yeah. This needs some work. The logic for checking for images in the last mini stack doesn't exist. We typically just modify the stack.vrt to get around this.
